_episodes/11-using-arrays-transformations.md

```diff
@@ -13,41 +13,45 @@ keypoints:
 ---
 
 ## Arrays
-An 'Array' is a list of values, represented in Refine by the use of square brackets containing a list of values surrounded by inverted commas and separated by commas. For example an array listing the days of the week would look like:
+An 'Array' is a list of values, represented in OpenRefine by the use of square brackets containing a list of values surrounded by inverted commas (or double quotes) and separated by commas. For example an array listing the Subjects column would look like:
 
-["Monday","Tuesday","Wednesday","Thursday","Friday","Saturday","Sunday"]
+["crystal structure", "clozapinium", "crystal structure", "molecular configuration", "hydrogen bonding", "supramolecular assembly", "Chemistry", "QD1-999"]
 
 Arrays can be sorted, de-duplicated, and manipulated in other ways in GREL expressions, but cannot appear directly in an OpenRefine cell. Arrays in OpenRefine are usually the result of a transformation. For example the ```split``` function takes a string, and changes it into an array based on a 'separator'. For example if a cell has the value:
```

Comment:
Change "appear" to "be stored"

Reply:
Change "the result of a transformation" to "the result of a transformation written with GREL"

Reply:
Done and Done